recipes-openxt/xctools/varstored-watch_git.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "Provides binary that restarts varstored on failure"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=4641e94ec96f98fabc56ff9cc48be14b"
+
+DEPENDS = " \
+    xen-tools \
+"
+
+RDEPENDS_${PN} = " \
+    varstored \
+"
+
+SRCREV = "${AUTOREV}"
+PV = "0+git${SRCPV}"
+
+S = "${WORKDIR}/git/varstored-watch"
+
+SRC_URI = " \
```

Comment:
and SRC_URI should be replaced by `require xctools.inc` for the final submission.  meta-openxt-externalsrc needs an update as well, which I can take care of.

Reply:
Roger. I'll wait for your testing first in case there are other changes.